encrypted-media-respec.html

```diff
@@ -3868,23 +3870,48 @@
             <li><p>If Key System messages contain information derived from a user identifier in a consistent manner, for example such that a portion of the initial Key System message for a specific content item does not change over time and is dependent on the user identifier, then this information could be used by any application to track the device or user over time.</p></li>
           </ul>
 
-          <p>In addition, if a Key System permits keys to be stored and to be re-used between origins, then it may be possible for two origins to collude and track a unique user by recording their ability to access a common key.</p>
+          <p>In addition, if a Key System permits keys or other data to be stored and to be re-used between origins, then it may be possible for two origins to collude and track a unique user by recording their ability to access a common key.</p>
           <p>Finally, if any user interface for user control of Key Systems presents data separately from data in HTTP session cookies [[COOKIES]] or persistent storage, then users are likely to modify site authorization or delete data in one and not the others.
             This would allow sites to use the various features as redundant backup for each other, defeating a user's attempts to protect his or her privacy.
           </p>
+          
+          <p>
+            In addition to the potential for sites and other third-parties to track users, the user agent implementer, CDM vendor, or device vendor could build a profile of the user's activities or interests, such as sites using the APIs defined in this specification that the user visits.
+            Such tracking would undermine the privacy protections provided by the rest of the web platform, especially those related to isolation of origins.
+          </p>
+          
+          <p>
+            Identifiers, such as <a def-id="distinctive-identifiers"></a>, are often obtained from a server operated or provided by the CDM vendor, such as via an <a href="#individualization">individualization</a> process.
+            The process may include providing client identifier(s), including <a def-id="distinctive-permanent-identifier-maybe-plural"></a>, to the server.
```

Comment:
I don't think we know very much about how people are doing individualization, so rather than "are often obtained" perhaps we should just say "may be obtained" (unless you have direct knowledge of people often doing it this way).